css/comp-page.css

```diff
@@ -1,48 +1,64 @@
-@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
-
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap");
 :root {
-    --primary-colour: #93dacd;
-    --secondary-colour: #e2e3e5;
+    --primary-colour: #00b594;
+    --secondary-colour: #d0f7f0;
+    --primary-font-colour: #fff;
+    --secondary-font-colour: #000;
+    --font: "Montserrat", sans-serif;
 }
 
 * {
     box-sizing: border-box;
     font-family: "Montserrat", sans-serif;
+    font-weight: 400;
 }
 
 .site-nav {
     position: fixed;
+    top: 0;
+    bottom: 0;
     margin-left: 5em;
-    margin-top: 0.5em;
+    margin-top: 7em;
+    overflow-y: auto;
 }
 
-.nav-heading {
-    font-size: 1.07rem;
+.nav-main-heading {
+    font-size: 1rem;
```

Comment:
Try to maintain consistencies in units for components. For eg. here font-size is in rem but rest all are in em.

Another good practice to have as suggested by Kushagra was to use rem. Just a good practice in terms of preference but definitely not compulsory though.